client-web/src/exlibris-app/menu/exlibris-menu-item.html

```diff
@@ -36,14 +37,12 @@
             }
         </style>
         <a href="[[link]]">
-        <div id="menu-item-body" class="item">
-            <slot name="icon"></slot>
-            <span id="menu-item-title" class="name">[[title]]</span>
-            <span id="menu-item-badge" class="count-badge">[[count]]</span>
-        </div>
+            <div id="menu-item-body" class="item">
+                <slot name="icon"></slot>
```

Comment:
Named slot is good when you have more than one slot. It's reduntant in this case.